### ISSUE TITLE
Do different FindVariableFeatures() selection methods affect a Control vs Treated analysis?

### ISSUE THREAD

**Issue by chatterjee89** [satijalab/seurat#3304]
Hi, I have a biologist's question for you. And I understand that this is a simple "It does" or "It doesn't" kind of question, but no one I have asked has been able to give me an answer and I've been running blind here.

When analyzing one sample, I have used the various methods that select variable features using the FindVariableFeatures() function. And each time, I mostly get faithful clustering with a few cells exchanged here and there between closely related/associated clusters. So, I choose one method randomly and just ignore the rest. However, I was wondering about the downstream effects of choosing the different methods when combining different samples together for plotting them as one, each having individually different variable features of their own because of different genotypes.

I find that this is a sensitive criteria given that the cells in question are technically very closely related and only differ developmentally - so in a sample with a specific treatment that has effectively caused de-differentiation(ish) of the various cell types, the general variability is quite low for that specific sample. And that when I put them together with the "Control" group, I am able to extract more specific "cell states" based on performing PCA on a combined list of variable features from both treated and Control samples. So far, this approach has really helped me find what I've been looking for in my experiments.

**TL;DR:** But please tell me if using the different methods (vst, disp and mvp) for finding variable features adversely affect this type of analysis, as I'm not sure I understand how combining samples with different (sequencing) technicalities would influence FindVariableFeatures() function?

**Comment by satijalab** [comment]
1. One advantage to using the SCTransform workflow (which automatically sets variable genes), is that genes are weighted by their amount of residual variance in downstream analyses. This way you can add in more genes and the results will not change as much (because the additional genes will have less weight)

2. If not using SCTransform, we do recommend VST.

3. While all methods will give slightly different results, and we do generally believe some work better than others (see above), there are no unique considerations to doing Ctrl vs Treatment analyses for this question
